src/character_controller.h

```diff
@@ -285,6 +285,7 @@ typedef struct character_s
     uint16_t                    bone_r_hand_end;
     int16_t                     weapon_id;
     int16_t                     weapon_id_req;
+    int                         currentAmmunitionInventoryItemId;
```

Comment:
naming code style... low_case

Reply:
Ok, I will change this in another commit.